### PR TITLE
Vendor in licence checker improvements, and fix an error message.

### PR DESCRIPTION
**Pull request by rswarbrick**

### CODE REVIEW

**Comment by rswarbrick** [comment]
(CI doesn't run until the patch is "ready for review" - aargh!) Assuming this fails in a helpful manner, I'll force-push to get rid of the final "bogus" patch and all should be ready for merge.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,6 +1,3 @@
-# Copyright lowRISC contributors.
```
Comment (lenary) [inline azure-pipelines.yml:1]: This file needs a licence, I think. If i excluded it in the config, that was potentially a mistake.

Comment (lenary) [reply azure-pipelines.yml:1]: Oh duuuur I see what you're doing.

**Comment by lenary** [comment]
Oh, It seems I missed adding the licence checker vendor directory to the excludes in `util/licence-checker.hjson`. Can you add that too (put it beside the other vendor directories).

File: azure-pipelines.yml (hunk 1 above)
Comment (rswarbrick) [reply azure-pipelines.yml]: That change was the last commit in the series (called "bogus"): the whole point was to check that it failed CI properly. Unfortunately, I couldn't get CI to run without marking as ready for a review :-/ Anyway, I've dropped that now, and all should be well!

**Comment by rswarbrick** [comment]
Now that I've removed the "bogus" commit, this should hopefully be mergeable.

**Comment by lenary** [comment]
The follow-up from me is in #2896.

**Pull request merged**